package.json

```diff
@@ -42,13 +42,18 @@
     "url": "https://github.com/Zooz/api-schema-builder/issues"
   },
   "homepage": "https://github.com/Zooz/api-schema-builder.git",
-  "author": "Gal Cohen",
+  "author": "Idan Tovi",
+  "contributors": [
+    "Igor Savin",
+    "Kobi Carmeli",
+    "Gal Cohen"
+  ],
   "dependencies": {
     "ajv": "^6.10.2",
     "clone-deep": "^4.0.1",
     "decimal.js": "^10.2.0",
     "js-yaml": "^3.13.1",
-    "json-schema-deref-sync": "^0.10.1",
+    "json-schema-deref-sync": "github:cvent/json-schema-deref-sync",
```

Comment:
Wouldn't it be safer to point to https://github.com/cvent/json-schema-deref-sync/releases/tag/v0.11.0 directly?

Reply:
they still haven't released it, that's why this PR is labeled as draft

Reply:
@kobik I don't mean the npm release; AFAIR, you can target specific git tags on GH repo.